### DISCUSSION TITLE
Local SVG in Next/Image component not loading when deployed on Netlify

### DISCUSSION THREAD

**Discussion by rileymacisaac** [vercel/next.js#23613] [Help] [answered]
Hey everyone,

When using next/image with an SVG logo, it works great during development. However, once I deploy to netlify it is a broken path.

**Code:**
`<Image src="/img/example-logo.svg" alt="Example" width={188} height={64} layout="fixed" priority="true" />`

**Results in this path:**
`/_next/image?url=%2Fimg%2Fexample-logo.svg&w=384&q=75`

Has anyone else encountered this?

**Comment by akellbl4** [comment, marked as answer]
As far as I know `next/image` just doesn't work with svg. By that I mean it's not able to optimize it. If you have only SVG in this example I could suggest just use `<img>` or set quality to 100%

**Reply by leerob** [reply]
Currently, there isn't a good reason to use next/image with SVGs. Since these are vector images, they're not optimized through next/image (which uses Squoosh + WASM). It would need to add support for SVG optimization with a tool like SVGO. 

I agree, just use a normal `img` tag or even an inline `svg` tag.

**Reply by rileymacisaac** [reply]
I was attempting to keep every image consistent by using next/image, but I guess it makes sense that next/image wouldn't support SVG. 

I've replaced it with an `img` as you've both mentioned and it works as expected.

Thanks!
